### ISSUE TITLE
Fix Request: MDCPutWithoutRemove false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#518]
```
@Component
@Aspect
public class MDCServiceCallAspect {
    private static final String KEY = "trace:service-call";

    @Around("@annotation(MDCServiceCall) && execution(* *(..))")
    public Object addMdcServiceCallVariables(ProceedingJoinPoint joinPoint) throws Throwable {
        MDCServiceCall annotation = ((MethodSignature) joinPoint.getSignature()).getMethod().getAnnotation(MDCServiceCall.class);

        if(!annotation.service().isBlank() && !annotation.operation().isBlank()) {
            MDC.put(KEY, annotation.service() + " - " + annotation.operation()); // <- false positive
        }

        try {
            return joinPoint.proceed();
        } finally {
            MDC.remove(KEY);
        }
    }
}
```

**Comment by jborgers** [comment]
The `put` does not need to be inside the try statement, yet, the `remove` must be.
